pages/linux/wol.md

```diff
@@ -0,0 +1,28 @@
+# wol
+
+> Client for sending Wake-on-LAN magic packets.
+> More information: <https://sourceforge.net/projects/wake-on-lan/>.
+
+- Send a WoL packet to a device:
+
+`wol {{mac_address}}`
+
+- Send a WoL packet to a device in another subnet based on its IP:
+
+`wol --ipaddr={{ip_address}} {{mac_address}}`
```

Comment:
This confuses me a bit.  Why would you need to specify the IP address and the MAC address?  In the first example it's quite obvious that you use the MAC address to specify what host to target, but in this one it seams that you use the IP to find the host instead.  I don't understand, then, the presence of both.  Same with the below example.